lms/static/sass/common-variables.scss

```diff
@@ -0,0 +1,72 @@
+/* Variables from simple_theme role start here */
+$link-color: #337233;
+
+$btn-primary-bg: #337233;
+$btn-primary-color: #FFF;
+$btn-primary-border-color: #337233;
+$btn-primary-hover-bg: #FFF;
+$btn-primary-hover-color: #337233;
+$btn-primary-hover-border-color: #337233;
+
+$btn-secondary-bg: #FFF;
+$btn-secondary-color: #337233;
+$btn-secondary-border-color: #337233;
+$btn-secondary-hover-bg: #337233;
+$btn-secondary-hover-color: #FFF;
+$btn-secondary-hover-border-color: #337233;
+
+$customize-sign-in-btn: true;
+$btn-sign-in-bg: #337233;
+$btn-sign-in-color: #FFF;
+$btn-sign-in-border-color: #337233;
+$btn-sign-in-hover-bg: #FFF;
+$btn-sign-in-hover-color: #337233;
+$btn-sign-in-hover-border-color: #337233;
+
+$customize-register-btn: true;
+$btn-register-bg: #FFF;
+$btn-register-color: #337233;
+$btn-register-border-color: #337233;
+$btn-register-hover-bg: #337233;
+$btn-register-hover-color: #FFF;
+$btn-register-hover-border-color: #337233;
+
+$customize-header-border: true;
+$header-border-top-width: 0px;
+$header-border-top-color: #545454;
+$header-border-bottom-width: 1px;
+$header-border-bottom-color: #545454;
+
+$customize-footer-border: true;
+$footer-border-top-width: 1px;
+$footer-border-top-color: #545454;
+$footer-border-bottom-width: 0px;
+$footer-border-bottom-color: #545454;
+
+$footer-link-color: #337233;
+$footer-color: #767676;
+
+$customize-logistration-action-btn: true;
+$btn-logistration-bg: #337233;
+$btn-logistration-color: #FFF;
+$btn-logistration-border-color: #337233;
+$btn-logistration-hover-bg: #FFF;
+$btn-logistration-hover-color: #337233;
+$btn-logistration-hover-border-color: #337233;
+$login-register-header-color:#337233;
+
+$homepage-bg-image: url("../images/hero_cover.jpg");
+
+$main-nav-item-border-bottom-color: #337233;
+$main-nav-item-hover-border-bottom-color: #337233;
+
+$course-nav-menu-color: #50991E;
+$course-nav-menu-border-bottom-color: #50991E;
+
+$account-settings-nav-border-bottom-color: #50991E;
+$account-settings-nav-border-hover-bottom-color: #50991E;
+
+$wrapper-preview-menu-color: #545454;
+
+$main-color: #337233;
+/* Variables from simple_theme role end here */
```

Comment:
Minor nit: I think you can remove the `/* Variables from simple_theme role end here */` comments from the files.